environment/OMAS_process.m

```diff
@@ -816,20 +822,15 @@
             
             delta_I = delta_I + p_jk*agents(j).memory_I;
             delta_i = delta_i + p_jk*agents(j).memory_i;
-            
-%             if p_jk
-%                 opt_dist.result.consenus{j_agent}.group_set{i_consensus} = union( opt_dist.result.consenus{j_agent}.group_set{i_consensus},...
-%                 opt_dist.result.consenus{k_agent}.group_set{i_consensus-1});
-%             end
         end
         
-        ratio = i / num_steps;
+        ratio = step / num_steps;
         Y = Y_prior + ratio*size_comp(i)*delta_I;
         y = y_prior + ratio*size_comp(i)*delta_i;
```

Comment:
I don't fully understand this ratio part. To me it makes sense if do:
Y = Y + (1/num_steps)*size_comp*delta_I
because then we do this 10 times, and we sort-of have 
Y = Y + size_comp*delta_I
which would be Y = Y + delta_I
but then delta_I weighted by the number of agents' information it includes.